packages/palette/src/elements/Skeleton/Skeleton.story.tsx

```diff
@@ -0,0 +1,13 @@
+import { storiesOf } from "@storybook/react"
+import React from "react"
+import { Skeleton } from "./Skeleton"
+
+storiesOf("Components/Skeleton", module).add("Basic", () => {
+  return (
+    <>
+      <Skeleton m={2} width={400} height={300} />
+      <Skeleton m={2} width={400} height={300} done />
```

Comment:
should `done` be the primary axis of behavior or would `loading` be more appropriate?

Reply:
The way I was thinking was that the skeleton is always loading by default because that's reason for existence and eventually receives a done which can reflect ready/error (either way stops the animation) — so I guess trying to avoid a default as false state.

Reply:
Cool, makes sense 👍